### PR TITLE
Create public-witnessing and meetings files for translation

### PR DESCRIPTION
**Pull request by mrtavor**
Added two new files public-witnessing.json and meetings.json. Also made a few fixes to other files.

### CODE REVIEW

File: src/shared/locales/en/meetings.json [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+ "tr_endWeek": "End week",
+ "tr_talkByVisitingSpeaker": "Delivered by visiting speaker",
+ "tr_openingSong": "Opening song",
+ "tr_secondSong": "Second song",
+ "tr_closingSong": "Closing song",
+ "tr_watchtowerStudy": "Watchtower study",
+ "tr_watchtowerStudyArticle": "Watchtower study article",
+ "tr_addOutgoingTalk": "Add outgoing talk",
+ "tr_weekendScheduleSaved": "The weekend schedule saved",
+ "tr_weekendScheduleSavedDesc": "Plan all the meetings that you want and then publish the corresponding schedules for others to view",
```
Comment (ux-git) [inline src/shared/locales/en/meetings.json:62]: Plan the meetings and publish their corresponding schedules for others to view.

File: src/shared/locales/en/public-witnessing.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  "tr_partnerNeeded": "Partner needed",
+  "tr_partnerNeededDesc": "Other publishers will see this and be able to arrange with you",
+  "tr_locationName": "Location name",
+  "tr_GeneralTimeRules": "General time rules",
+  "tr_maxPublisherLabel": "Max. publishers per shift ",
+  "tr_whereGetCart": "Where to get a cart",
+  "tr_PWShifts": "Public witnessing shifts",
+  "tr_selectDays": "Select days",
+  "tr_everyDay": "Every day",
+  "tr_daysShifts": "shifts {{ dayName }}",
```
Comment (ux-git) [inline src/shared/locales/en/public-witnessing.json:15]: @mrtavor perhaps "Shifts: {{ dayName }}"?

File: src/shared/locales/en/public-witnessing.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  "tr_editLocations": "Edit locations",
+  "tr_PW": "Public witnessing",
+  "tr_locationCart": "Cart on {{ locatinName }}",
+  "tr_maxPublisherShift": "Max. publishers per shift: {{ maxPublisherCount }}",
+  "tr_partnerNeeded": "Partner needed",
+  "tr_partnerNeededDesc": "Other publishers will see this and be able to arrange with you",
+  "tr_locationName": "Location name",
+  "tr_GeneralTimeRules": "General time rules",
+  "tr_maxPublisherLabel": "Max. publishers per shift ",
+  "tr_whereGetCart": "Where to get a cart",
```
Comment (ux-git) [inline src/shared/locales/en/public-witnessing.json:11]: @rhahao what is your opinion? How to call a place where the stand is stored _(e.g. at a brother or at a Kingdom Hall_)? What short and generic phrasing we can use here to include everything (cart, literature table, something else) and be simple? For example, pick-up point, storage point, storage location? Any better options?

**Review by ux-git — CHANGES_REQUESTED** [review]

File: src/shared/locales/en/meetings.json [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+ "tr_addWeekManuallyDesc": "<p>It is recommended to add a new week only if you have no opportunity to import the meeting materials from .epub file or from jw.org. Otherwise, the added week will be empty and you will have to fill in all the meeting parts manually.</p><p>The new week will be added as the newest week in your list.</p>",
+ "tr_unsavedChanges": "You have unsaved changes",
+ "tr_unsavedChangesDesc": "Do you want to leave the page without saving the edits? You will lose all unsaved changes.",
+ "tr_saveChanges": "Save the changes",
+ "tr_leavePage": "Leave the page",
+ "tr_autofillMM": "Autofill midweek meeting",
+ "tr_autofillMMDesc": "Select which schedules you want to autofill with qualified participants.",
+ "tr_startWeek": "Start week",
+ "tr_endWeek": "End week",
+ "tr_talkByVisitingSpeaker": "Delivered by visiting speaker",
```
[the remark this answers is not in the document]
Comment (mrtavor) [reply src/shared/locales/en/meetings.json:54]: "Talk by visiting speaker" sounds much easier and better to me

File: src/shared/locales/en/meetings.json [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+ "tr_clearMultipleWMDesc": "Select which schedules you want to clear. The entered information will be removed, but the weekly templates will remain available.",
+ "tr_clearSelectedWeeks": "Clear selected weeks",
+ "tr_addWeekManually": "Add a week manually",
+ "tr_addWeekManuallyDesc": "<p>It is recommended to add a new week only if you have no opportunity to import the meeting materials from .epub file or from jw.org. Otherwise, the added week will be empty and you will have to fill in all the meeting parts manually.</p><p>The new week will be added as the newest week in your list.</p>",
+ "tr_unsavedChanges": "You have unsaved changes",
+ "tr_unsavedChangesDesc": "Do you want to leave the page without saving the edits? You will lose all unsaved changes.",
+ "tr_saveChanges": "Save the changes",
+ "tr_leavePage": "Leave the page",
+ "tr_autofillMM": "Autofill midweek meeting",
+ "tr_autofillMMDesc": "Select which schedules you want to autofill with qualified participants.",
```
[the remark this answers is not in the document]
Comment (mrtavor) [reply src/shared/locales/en/meetings.json:51]: 👍

File: src/shared/locales/en/meetings.json [hunk 6]
```diff
@@ -0,0 +1,92 @@
+{
+ "tr_display": "Display",
+ "tr_hall": "Hall {{ hallNumber }}", 
+ "tr_yourUserRecordOfTheList": "To see your name in the list, make sure your person record exists on the All persons page.",
```
[the remark this answers is not in the document]
Comment (rhahao) [reply src/shared/locales/en/meetings.json:4]: Hello @ux-git @mrtavor! For any similar case like this one, just wrap the text with an anchor tag with # in `href`:

```json
"tr_yourUserRecordOfTheList": "To see your name in the list, make sure your person record exists on the <a href='#'>All persons page</a>."
```
This `href` will be updated dynamically later.

File: src/shared/locales/en/public-witnessing.json (hunk 3 above)
Comment (rhahao) [reply src/shared/locales/en/public-witnessing.json]: @ux-git, @mrtavor: Let’s consider refining the naming later. In that way, we can merge this PR at this time.

Comment (ux-git) [reply src/shared/locales/en/public-witnessing.json]: @rhahao sure 👍

**Pull request closed without merging**